README.md

```diff
@@ -1,13 +1,36 @@
 # Chainblocks examples
 
-Example games and tutorials using [chainblocks](https://github.com/fragcolor-xyz/chainblocks).
+[![license](https://img.shields.io/github/license/fragcolor-xyz/chainblocks-examples)](./LICENSE)
+[![docs](https://img.shields.io/badge/docs-API-blueviolet)](https://docs.fragcolor.xyz/)
 
-## Tutorials
+This repository is a collection of example games and tutorials using the [Chainblocks](https://github.com/fragcolor-xyz/chainblocks) programming language.
+
+Chainblocks is a lisp and hence is extremely powerful as well as fun to learn.
+
+These tutorials are designed to help you learn practical Chainblocks programming step-by-step. Each tutorial covers a few langauge features, blocks, and functions in the context of building a game from scratch.
+
+## Installation
+
+[Set up your environment](https://docs.fragcolor.xyz/contribute/code/getting-started/) and [build Chainblocks](https://docs.fragcolor.xyz/contribute/code/building-chainblocks/) before working through the tutorials given in this repository.
+
+## Usage
+
+Though there are no hard pre-requisites for these tutorials, if you're new to programming you should read the [Chainblocks primer](https://learn.fragcolor.xyz/how-to/chainblocks-primer/) first.
+
+It's also recommended to go through the [Chainblocks architecture](https://docs.fragcolor.xyz/architecture-guides/chainblocks) guide to understand what's going on under the hood.
```

Comment:
Added TODO page for Chainblocks architecture guide link with [this commit](https://github.com/fragcolor-xyz/chainblocks/pull/238/commits/2db79b473ee45e88eeebd10a624e779c4616b6f1)